data/docs/gcp-monitoring/gke-tracing.mdx

```diff
@@ -0,0 +1,1190 @@
+
+Use the appropriate export URL in the script. The export url will receive all the traces captured by Pixie.
+
+<Tabs>
+<TabItem value="self-hosted-on-gke-cluster" label="For self-hosted SigNoz in GKE" default>
+
+```sh
+	url='<endpoint-ip-of-otel-svc>':4317,
+	insecure=True
+```
+OR
+
+```sh
+	url='<self-host-signoz-otel-collector-service-fqdn>':4317,
+	insecure=True
+```
+
+</TabItem>
+<TabItem value="self-host-signoz-in-vm" label="For self-hosted SigNoz in VM">
+
+```sh
+	url='<endpoint-ip-of-vm>:4317',
+	insecure=True
+```
+
+</TabItem>
+</Tabs>
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/pixie-config-1-self-hosted.webp"
+        alt="Pixie Configuration"
+    />
+<figcaption>
+<i>
+Pixie Configuration
+</i>
+</figcaption>
+</figure>
+
+
+**Step 3:** To test the configuration, deploy a sample application in the cluster to view its traces in the SigNoz.
+
+```sh
+kubectl run -n signoz my-hello –image=nginx –port=80
+```
+
+### Visualize the Traces in SigNoz self-hosted
+
+To visualize the traces, log into the SigNoz account and navigate to the traces section.
+
+<Tabs>
+<TabItem value="self-hosted-on-gke-cluster" label="For self-hosted SigNoz in GKE" default>
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/signoz-self-hosted-gke-ebpf.webp"
+        alt="Traces in SigNoz Dashboard"
+    />
+<figcaption>
+<i>
+Traces in SigNoz Dashboard (self-hosted on GKE Cluster)
+</i>
+</figcaption>
+</figure>
+
+</TabItem>
+
+<TabItem value="self-host-signoz-in-vm" label="For self-hosted SigNoz in VM">
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/signoz-sel-hosted-ebpf.webp"
+        alt="Traces in SigNoz Dashboard"
+    />
+<figcaption>
+<i>
+Traces in SigNoz Dashboard (self-hosted on a VM)
+</i>
+</figcaption>
+</figure>
+
+</TabItem>
+</Tabs>
+
+## APM and Distributed Tracing
+
+For application-level tracing, you can use the OpenTelemetry SDKs integrated with your application. These SDKs automatically collect and forward traces to the central collector.
+
+Please refer to our [SigNoz Tutorials](https://signoz.io/docs/instrumentation/) or [Blog](https://signoz.io/blog/) to find information on how to instrument your application like Spring, FastAPI, NextJS, Langchain, Node.js, Flask, Django, etc.
+
+### Sample Python Application 
+
+We will use a [sample flask app](https://github.com/SigNoz/sample-flask-app).
+One may need to add packages (in `requirements.txt`) for enabling instrumentation.
+
+```txt
+Flask==3.0.0 
+pymongo==3.12.1 
+requests==2.26.0 
+opentelemetry-api==1.22.0 
+opentelemetry-distro==0.43b0 
+opentelemetry-instrumentation==0.43b0 
+opentelemetry-exporter-otlp==1.22.0 
+``` 
+
+Install those packages in dockerfile configuration.
+
+```Dockerfile 
+… 
+# install dependencies
+RUN pip install -r requirements.txt
+RUN opentelemetry-bootstrap --action=install…  
```

Comment:
'...' should be in new line not after install keyword, wanted to indicated that we've truncated  Dockerfile, it is not full file

Reply:
fixed